src/server/views/tag-env/template.njk

```diff
@@ -0,0 +1,15 @@
+{%- from "govuk/components/tag/macro.njk" import govukTag -%}
+
+{%- switch env %}
+    {% case "test" %}
+      {% set classes = "govuk-tag--yellow" %}
+    {% case "production" %}
+      {% set classes = "govuk-tag--red" %}
+    {% default %}
+      {% set classes = "govuk-tag--grey" %}
+  {% endswitch -%}
+
+  {{ govukTag({
+    text: env | capitalize,
+    classes: classes
+  }) }}
```

Comment:
```suggestion
  {% case "test" %}
    {% set classes = "govuk-tag--yellow" %}
  {% case "production" %}
    {% set classes = "govuk-tag--red" %}
  {% default %}
    {% set classes = "govuk-tag--grey" %}
{% endswitch -%}

{{ govukTag({
  text: env | capitalize,
  classes: classes
}) }}
```

Reply:
Same 😭

Reply:
same? are you sure? did you pull latest?

Reply:
Ahh sorry, no I mean I also wish we had a linter for it! 😅